pygmt/src/grdclip.py

```diff
@@ -0,0 +1,89 @@
+"""
+grdclip - Clip a grid.
+"""
+
+import xarray as xr
+from pygmt.clib import Session
+from pygmt.helpers import (
+    GMTTempFile,
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+
+
+@fmt_docstring
+@use_alias(
+    G="outgrid",
+    R="region",
+    Sa="above",
+    Sb="below",
+    Si="interval",
+    Sr="old",
+    V="verbose",
+)
+@kwargs_to_strings(
+    R="sequence",
+    Sa="sequence",
+    Sb="sequence",
+    Si="sequence",
+    Sr="sequence",
+)
+def grdclip(grid, **kwargs):
+    r"""
+    Sets specific values, or values above/below a set number, to a given value.
+
+    Produce a new ``outgrid`` file clipped version of``grid``.
+
+    The parameters ``above`` and ``below`` allow for a given value to be set
+    for values above or below a set amount, respectively. This allows for
+    extreme values in a grid, such as points below a certain depth when
+    plotting Earth relief, to all be set to the same value.
+
+    Full option list at :gmt-docs:`grdclip.html`
+
+    {aliases}
+    Parameters
```

Comment:
Add an extra newline here.
```suggestion
    {aliases}

    Parameters
```